website/pages/docs/concepts/seal.mdx

```diff
@@ -49,18 +56,19 @@ This process is stateful: each key can be entered via multiple mechanisms
 on multiple computers and it will work. This allows each shard of the master
 key to be on a distinct machine for better security.
 
-Once a Vault is unsealed, it remains unsealed until one of two things happens:
+Once a Vault node is unsealed, it remains unsealed until one of two things happens:
 
 1. It is resealed via the API (see below).
 
 2. The server is restarted.
 
+3. Something sufficiently bad happens at the storage layer that Vault decides
```

Comment:
Maybe a slightly less colloquial wording here would be better.

Reply:
"Vault's storage layer encounters an unrecoverable error"?

Reply:
yep, sounds good